packages/main/src/Popup.ts

```diff
@@ -188,6 +189,18 @@ abstract class Popup extends UI5Element {
 	@property({ defaultValue: "" })
 	accessibleNameRef!: string;
 
+	/**
+	 * Allows setting a custom role. Available options are:
+	 * <code>Dialog</code>, <code>None</code> and <code>AlertDialog</code>.
```

Comment:
write those entries in a list
for example, see the docs here https://sap.github.io/ui5-webcomponents/playground/components/Avatar/